File: TransCoder_tokenized_test_set_functions/python/MINIMUM_OPERATION_MAKE_ELEMENTS_EQUAL_ARRAY.py
def minOperation ( arr, n ) :
    Hash = defaultdict ( lambda : 0 )
    for i in range ( 0, n ) :
        Hash [ arr [ i ] ] += 1
    max_count = 0
    for i in Hash :
        if max_count < Hash [ i ] :
            max_count = Hash [ i ]
    return n - max_count

File: TransCoder_tokenized_test_set_functions/java/MINIMUM_OPERATION_MAKE_ELEMENTS_EQUAL_ARRAY.java
public static int minOperation ( int arr [ ], int n ) {
  HashMap < Integer, Integer > hash = new HashMap < Integer, Integer > ( );
  for ( int i = 0;
  i < n;
  i ++ ) if ( hash . containsKey ( arr [ i ] ) ) hash . put ( arr [ i ], hash . get ( arr [ i ] ) + 1 );
  else hash . put ( arr [ i ], 1 );
  int max_count = 0;
  Set < Integer > s = hash . keySet ( );
  for ( int i : s ) if ( max_count < hash . get ( i ) ) max_count = hash . get ( i );
  return ( n - max_count );
}